jetstream/leafnodes.md

```diff
@@ -9,7 +9,7 @@ The general issue with multiple, independent JetStreams, accessible from the sam
 As an example, consider a leafnode with a non-clustered JetStream in each server. 
 You connect to one of them, but which JetStream responds when you use the JetStream API `$JS.API.>` ?
 
-To disambiguate that, the option `domain` was added to the JetStream configuration block. 
+To disambiguate between servers, the option `domain` was added to the JetStream configuration block. 
 When using it, follow these rules:
```

Comment:
Do we need to add the `domain` config option somewhere else in the docs; Leafnode config? Server config?

Reply:
I haven't really found s place for that